copilot/app.py

```
@@ -0,0 +1,34 @@
+                metadata={"labels": {"app": name}},
+                spec=client.V1PodSpec(
+                    containers=[client.V1Container(name=name, image=image)]
+                )
+            ),
+        )
+    )
+
+    try:
+        v1.create_namespaced_deployment(namespace="default", body=deployment)
+        return {"message": f"Deployment {name} created successfully"}
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)